---
output: html_document
---

```{r setup, include=FALSE, warning=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```
# Final Project
Yuki Gaudreault

BI 410/510: Data Management and Visualization

Dr. Lauren Hallett

December 8, 2021

## I. Introduction

In eukaryotic cells, DNA is highly organized to form a complex 3D genome structure (Zheng & Xie, 2019). This structure is of great interest because when defective, it can lead to development abnormalities and human disease (Zheng & Xie, 2019). Folded chromosomes are called chromatin domains and can be either active or inactive, meaning the genetic code in the area is or is not read (transcribed), respectively (Simonis, 2006). It is known that eukaryotic genomes are organized in differently sized chromatin domains, including topologically associating domains (TADs) (Noma, 2017; Tanizawa et al., 2017). TADs represent genome-organizing units which organize active and inactive chromatin domains and therefore coregulate transcription (Dixon et al., 2012; Tanizawa et al., 2017). However, it is still unknown how different types of chromatin domains are formed and regulated. 

The in situ Hi-C method has been developed for the human genome to elucidate the 3D genome structure by sequencing DNA strands that are bound together by proteins (Lieberman-Aiden et al., 2009). By mapping all of these interactions between different loci, we can determine which structures are close together and therefore the 3D structure of the genome (Lieberman-Aiden et al., 2009). It has been established from Hi-C experiments on the human genome that the greater the distance between loci of the same chromosome, the less the probability that they interact (reduced intrachromosomal interactions with distance) (Lieberman-Aiden et al., 2009).

The in situ Hi-C method has been also been applied to fission yeast cells to show that condensin and cohesin each form large chromatin domains (~ 500 kb) and small domains (~ 50 kb), respectively (Lieberman-Aiden et al., 2009; Tanizawa et al., 2017). Condensin protein complexes are known to be crucial in the structural formation of mitotic chromosomes (Hudson et al., 2009) while cohesin protein complexes are necessary for sister chromatid cohesion (Nasmyth & Haering, 2009). In terms of their domains, it was found that condensin domains are chromosomal compaction units that are required for proper chromosome segregation, whereas cohesin domains are promoted during interphase (Tanizawa et al., 2017). All of this has been found using Hi-C sequencing data of wild type strains compared against strains with mutated condensin and cohesin complexes (Tanizawa et al., 2017). Though mechanisms that drive the regulation of cohesin binding have been elucidated, it is still unclear how condensin domains are formed and regulated.

To investigate the mechanism of condensin domain formation, the Noma lab at the University of Oregon has performed yeast two-hybrid screening (Fields & Song, 1989) to identify 25 proteins that interact with condensin. Among these, 9 genes are essential for viability. These include sds3 which encodes a histone deacetylase complex I (HDAC I) subunit (Nicholas et al., 2007). HDAC I preferentially deacetylates histones at specific intergenic regions that have promoters (Nicholas et al., 2007). During the past year, I have constructed conditional knock-out strains for these essential genes using the auxin-inducible degradation system which conditionally allows for the complete removal of target proteins (Kanke et al., 2011). I have also performed an in situ Hi-C experiment on the sds3 deletion strain to be sequenced at the Genomics & Cell Characterization Core Facility (Lieberman-Aiden et al., 2009). Thus, the data available has expanded to Hi-C sequencing models of strains with mutated proteins that are known to interact with condensin. Bioinformatics analysis of this sequencing data will show how frequently genomic loci interact across the fission yeast genome. 

In this project, I propose to further analyze these sequencing results by combining wild type and Δsds3 sequencing data to determine the role that it plays in both inter and intrachromosome organization in fission yeast. This data visualization will help advance our understanding of how the 3D genome is organized and regulated and reveal novel mechanisms regulating condensin domains that are essential for proper chromosome segregation. Specifically, I will address the following questions:

- What role does Sds3 have in genome organization? 

- Does Sds3 have a role in regulating condensin domains?

- How does Sds3 affect the change association strength over distance?

I predict that my analysis will show that Sds3 has a significant positive role in regulating condensin domains (demonstrated by TADs), and that more distant locations within chromosomes will have reduced association strength regardless of Sds3’s presence. 

## II. Approach/Methods
### Dataset Identification
The data I will be using is the result of the sequencing of Hi-C samples at the Genomics & Cell Characterization Core Facility:

https://www.dropbox.com/sh/r0buz9fylmr0rwn/AACczw5mdRdfytfiHLDHMF4Ka?dl=0 

The data is in the form of matrices, which can be converted to a data frame that contains the two interacting genome locations and association strength (score) for each genome-genome interaction. 

### Workflow Plan
#### QAQC:
(For both wild type and Δsds3 sequencing data)

1. Convert matrices to a data frame

2. Remove NA scores

3. Split columns describing interacting gene locations into chromosome name, gene start position, and gene end position

```{r, include = FALSE, warning=FALSE}
library(tidyverse)
demo(plotmath)

## Store file names so they're easier to use
file_name <- "Control_Aux-6hr_HiC_Single-M_100kb.rds"
file_name1 <- "sds3-3mAID_Aux-6hr_HiC_Single-M_100kb.rds"

## QAQC

## Function for QAQC
convert_to_dataframe <- function(file_name){
  ## convert matrices to data frame
  map <- readRDS(file_name)
  r <- rownames(map)
  df <- as.data.frame(map) %>% mutate(bin1=r) %>% tidyr::gather(key="bin2", value="score", -bin1)
  ## remove NA scores
  df <- df %>% filter(!is.na(score))
  ## split columns describing interacting gene locations into chromosome name, gene start position, and gene end position
  df <- df %>% tidyr::separate(bin1, c("chr1", "start1", "end1"), ":", convert=TRUE, extra="merge")
  df <- df %>% tidyr::separate(bin2, c("chr2", "start2", "end2"), ":", convert=TRUE, extra="merge")
  df
}

## Make control dataframe
Control_hic <- convert_to_dataframe(file_name)
## Make Δsds3 dataframe
sds3_hic <- convert_to_dataframe(file_name1)
## Change name of Δsds3 score so it's distinguishable from Control's
names(sds3_hic)[7] <- "score1"
## Make dataframe with score difference of Control-Δsds3
Control_sds3_hic <-right_join(Control_hic, sds3_hic) %>%
  mutate(newscore=score-score1)
```

#### Part I: Heat map workflow:
1. Use geom tile with a color gradient to make a heat gradient of all the genome interactions (graph location of chr1 vs chr2) for both wild type and Δsds3 sequencing data

2. Join wild type and Δsds3 sequencing data frames

3. Mutate to make column of association score (strength) difference (wild type - Δsds3)

4. Use geom tile with a color gradient to make a heat gradient of the genome interactions in wild type and Δsds3 individually, as well as the difference in the genome interactions between wild type and Δsds3 sequencing data (graph location of chr1 vs chr2)

#### Part II: Association strength vs distance:
1. Filter control sequencing data to keep only intrachromosome interactions

2. Mutate to make columns of distance between interacting loci (start1 + end1 - start2 - end2)/2)

3. Select to keep only chromosome number, distance, and score

4. Group by chromosome and distance and summarize to find the average association strength for each distance per chromosome

5. Repeat #1-4 for Δsds3 sequencing data

6. Join these dataframes which show average association per distance

7. Gather to make new column distinguishing strains

8. Plot association score vs distance in a lineplot using ggplot, facet to make a different plot for each chromosome

#### Part III: Association strength vs distance depending on Sds3 presence:
1. Bind control and sds3 sequencing data with mutations with a column of strain name

2. Spread to make columns of scores labeled with the strain name

3. Plot association strength of wild type vs of delta-sds3 strains at each locus using scatterplot

## III. Results

NOTE: 'Association Strength' corresponds to the sequenced read number and therefore has no unit

```{r, message = FALSE, warning=FALSE}
## HEAT MAP

## Align locations in genome so chromosomes in heatmap are shown in order instead of overlapping
## (add 5600000bp to chrII locations, add 4500000+5600000bp to chrIII locations)

## Isolate chrI's of first in interacting pair (chr1)
IControl_sds3_I <- Control_sds3_hic %>%
  filter(chr1 == "I") %>%
  ## Mutate to create new adjusted locations
  mutate(start1new = start1) %>%
  mutate(end1new = end1)

## Isolate chrII's of first in interacting pair (chr1)
IControl_sds3_II <- Control_sds3_hic %>%
  filter(chr1 == "II") %>%
  ## Mutate to create new adjusted locations
  mutate(start1new = start1+5600000) %>%
  mutate(end1new = end1+5600000)

## Isolate chrIII's of first in interacting pair (chr1)
IControl_sds3_III <- Control_sds3_hic %>%
  filter(chr1 == "III") %>%
  ## Mutate to create new adjusted locations
  mutate(start1new = start1+5600000+4500000) %>%
  mutate(end1new = end1+5600000+4500000)

## Join all adjusted of first in interacting pair (chr1)
IControl_sds3 <- rbind(IControl_sds3_I, IControl_sds3_II, IControl_sds3_III)

## Isolate chrI's of second in interacting pair (chr2)
IIControl_sds3_I <- IControl_sds3 %>%
  filter(chr2 == "I") %>%
  ## Mutate to create new adjusted locations
  mutate(start2new = start2) %>%
  mutate(end2new = end2)

## Isolate chrII's of second in interacting pair (chr2)
IIControl_sds3_II <- IControl_sds3 %>%
  filter(chr2 == "II") %>%
  ## Mutate to create new adjusted locations
  mutate(start2new = start2+5600000) %>%
  mutate(end2new = end2+5600000)

## Isolate chrIII's of second in interacting pair (chr2)
IIControl_sds3_III <- IControl_sds3 %>%
  filter(chr2 == "III") %>%
  ## Mutate to create new adjusted locations
  mutate(start2new = start2+5600000+4500000) %>%
  mutate(end2new = end2+5600000+4500000)

## Join all adjusted of second in interacting pair (chr2)
IIControl_sds3 <- rbind(IIControl_sds3_I, IIControl_sds3_II, IIControl_sds3_III)

## Join aligned dataframes
Control_sds3_aligned <- right_join(IControl_sds3, IIControl_sds3)

## Make loci interaction heatmap of Control
ggplot(Control_sds3_aligned, aes(x = start1new, y=start2new, fill = score)) + 
  geom_tile() +
  ## Gradient limits adjusted to more easily see weaker interactions
  scale_fill_gradient(low="white", high="red", limits=c(0,2000), na.value='white', oob = scales::squish, name = "Interaction Strength") +
  ## Reverse y axis, remove gap btw map and axis, scale for each chromosome
  scale_y_continuous(trans = "reverse", expand = c(0, 0), breaks=c(2800000, 7850000, 11300000),
        labels=c("Chr I", "Chr II", "Chr III")) +
  ## Move x axis to top, remove gap btw map and axis, scale for each chromosome
  scale_x_continuous(position = "top", expand = c(0, 0), breaks=c(2800000, 7850000, 11300000),
        labels=c("Chr I", "Chr II", "Chr III")) +
  geom_hline(yintercept=5600000, size=0.5, color="grey", linetype = "dashed") +
  geom_hline(yintercept=10100000, size=0.5, color="grey", linetype = "dashed") +
  geom_vline(xintercept=5600000, size=0.5, color="grey", linetype = "dashed") +
  geom_vline(xintercept=10100000, size=0.5, color="grey", linetype = "dashed") +
  xlab("Gene Location 2 (bp)") + ylab("Gene Location 1 (bp)") +
  ## Add caption
  labs(caption = "Figure I.1. Heat gradient of genome interactions in wild type strains") +
  ## Remove ticks, left justify caption
  theme(axis.ticks = element_blank(),
        plot.caption=element_text(hjust = 0))

## Make loci interaction heatmap of Δsds3
ggplot(Control_sds3_aligned, aes(x = start1new, y=start2new, fill = score1)) + 
  geom_tile() +
  ## Gradient limits adjusted to more easily see weaker interactions
  scale_fill_gradient(low="white", high="blue", limits=c(0,2000), na.value='white', oob = scales::squish, name = "Interaction Strength") +
  ## Reverse y axis, remove gap btw map and axis, scale for each chromosome
  scale_y_continuous(trans = "reverse", expand = c(0, 0), breaks=c(2800000, 7850000, 11300000),
        labels=c("Chr I", "Chr II", "Chr III")) +
  ## Move x axis to top, remove gap btw map and axis, scale for each chromosome
  scale_x_continuous(position = "top", expand = c(0, 0), breaks=c(2800000, 7850000, 11300000),
        labels=c("Chr I", "Chr II", "Chr III")) +
  geom_hline(yintercept=5600000, size=0.5, color="grey", linetype = "dashed") +
  geom_hline(yintercept=10100000, size=0.5, color="grey", linetype = "dashed") +
  geom_vline(xintercept=5600000, size=0.5, color="grey", linetype = "dashed") +
  geom_vline(xintercept=10100000, size=0.5, color="grey", linetype = "dashed") +
  xlab("Gene Location 2 (bp)") + ylab("Gene Location 1 (bp)") +
  ## Add caption
  labs(caption = "Figure I.2. Heat gradient of genome interactions in delta-sds3 strains") +
  ## Remove ticks, left justify caption
  theme(axis.ticks = element_blank(),
        plot.caption=element_text(hjust = 0))

## Make loci interaction heatmap of Control-Δsds3
ggplot(Control_sds3_aligned, aes(x = start1new, y=start2new, fill = newscore)) + 
  geom_tile() +
  ## Gradient limits adjusted to more easily see weaker interactions
  scale_fill_gradient2(low="blue", high="red", mid = "white", limits=c(-2000,2000), na.value='white', oob = scales::squish, name = "Difference in \nInteraction Strength") +
  ## Reverse y axis, remove gap btw map and axis, scale for each chromosome
  scale_y_continuous(trans = "reverse", expand = c(0, 0), breaks=c(2800000, 7850000, 11300000),
        labels=c("Chr I", "Chr II", "Chr III")) +
  ## Move x axis to top, remove gap btw map and axis, scale for each chromosome
  scale_x_continuous(position = "top", expand = c(0, 0), breaks=c(2800000, 7850000, 11300000),
        labels=c("Chr I", "Chr II", "Chr III")) +
  geom_hline(yintercept=5600000, size=0.5, color="grey", linetype = "dashed") +
  geom_hline(yintercept=10100000, size=0.5, color="grey", linetype = "dashed") +
  geom_vline(xintercept=5600000, size=0.5, color="grey", linetype = "dashed") +
  geom_vline(xintercept=10100000, size=0.5, color="grey", linetype = "dashed") +
  xlab("Gene Location 2 (bp)") + ylab("Gene Location 1 (bp)") +
  ## Add caption
  labs(caption = "Figure I.3. Heat gradient of the difference in genome interactions between wild type and delta-sds3 strains\nBlue dots show increased interactions at the mating type locus when Sds3 is not present") +
  ## Remove ticks, left justify caption
  theme(axis.ticks = element_blank(),
        plot.caption=element_text(hjust = 0),
        legend.title=element_text(vjust = 2))
```

```{r, message = FALSE, warning=FALSE}
## ASSOCIATION STRENGTH VS DISTANCE

## Make control loci distance dataframe
Control_distance <- Control_hic %>%
  ## Keep only intra-chromosome interactions
  filter(chr1==chr2) %>%
  ## Mutate to make columns of distance between interacting loci
  mutate(distance = abs(start1 + end1 - start2 - end2)/2) %>%
  ## Select to keep only distance and score
  select(chr1, distance, score) %>%
  ## Group by chromosome and distance to find the average association strength for each
  group_by(chr1, distance) %>% 
  summarize(average = mean(score)) %>% 
  ungroup()

## Make Δsds3 loci distance dataframe
sds3_distance <- sds3_hic %>%
  ## Keep only intra-chromosome interactions
  filter(chr1==chr2) %>%
  ## Mutate to make columns of distance between interacting loci
  mutate(distance = abs(start1 + end1 - start2 - end2)/2) %>%
  ## Select to keep only distance and score
  select(chr1, distance, score1) %>%
  ## Group by chromosome and distance to find the average association strength for each
  group_by(chr1, distance) %>% 
  summarize(average1 = mean(score1)) %>% 
  ungroup()

## Join distance dataframes
Control_sds3_distance <- right_join(Control_distance, sds3_distance) %>%
  ## Make new column distinguishing strains
  gather(key = Strain, value = average, c("average", "average1"))

## Distance vs score lineplot
ggplot(Control_sds3_distance, aes(x=distance/1000000, y=average, color=Strain)) +
  geom_line(size = 1) + 
  xlab("Distance (Mb)") + ylab("Average Interaction Strength") +
  ## Use a log10 scale
  scale_x_log10() +
  scale_y_log10() +
  ## Add caption and title for top axis
  labs(caption = "Figure II.1. Association strength vs loci distance in each chromosome for wild type and delta-sds3 strains", title="Chromosome") +
  ## Make dark axis, left justify caption, set top axis size and alignment
  theme(axis.line = element_line(colour = "black"), plot.caption=element_text(hjust = 0), plot.title=element_text(size=11, hjust=0.5)) +
  ## Set legend labels and color palette
  scale_color_manual(labels = c("Wild type", "delta-sds3"), values=c("red", "blue")) +
  ## Create a different plot for each chromosome
  facet_wrap(~ chr1)
```

```{r, message = FALSE, warning=FALSE}
## ASSOCIATION STRENGTH VS DISTANCE DEPENDING ON SDS3 PRESENCE
## Remake sds3_hic so score doesn't have different name
sds3_hic <- convert_to_dataframe(file_name1)
## Make dataframe with columns for both control and sds3 scores for each locus
score_comparison <- rbind(Control_hic %>% mutate(sample = "control"), sds3_hic %>% mutate(sample = "sds3")) %>% tidyr::spread(key=sample, value=score)

## Make dataframe with just mating type locus points
matloci_comparison <- score_comparison %>%
  filter(between(start1, 2000000, 2200000)) %>% 
  mutate(matlocus=TRUE)

## Add mating type ID column to original dataframe
score_comparison <- score_comparison %>%
  mutate(matlocus=FALSE)

## Bind mat locus ID dataframe to score comparison dataframe
score_comparison <- rbind(score_comparison, matloci_comparison)

## Scatterplot of association strength of wild type vs of delta-sds3 strains at each locus
ggplot(score_comparison, aes(x=sds3, y=control, color=matlocus)) +
  geom_point() + 
  ## Log10 scale
  scale_x_log10(labels = function(x) format(x, scientific = FALSE)) + 
  scale_y_log10(labels = function(x) format(x, scientific = FALSE)) +
  ## Add caption
  labs(caption = "Figure III. Association strength of wild type vs of delta-sds3 strains at each locus\nMating type locus is highlighted in blue to show trends in this locus") +
  ## Align caption to the left
  theme(plot.caption=element_text(hjust = 0)) +
  ## Rename axes
  xlab("delta-sds3 Association Strength") + ylab("Wild Type Association Strength") +
  scale_color_manual(name = "Locus", labels = c("Other", "Mating Type\nLocus"), values = c("red", "blue"))
```

## IV. Discussion

Overall, it seems that Sds3 does not have a great role in regulating condensin domains but has an important role in regulating both inter and intrachromosome interactions. The lack of TADs in Figure I.3 shows that Sds3 does not take part in the process of the formation of condensin domains. However, the weaker interactions at the mat loci (in blue, Figure I.3 and Figure III) show that Sds3 is important in regulating these gene loci. On the other hand, Figure II shows that the lack of the Sds3 protein results in reduced intrachromosome interactions, meaning it is important in maintaining a stable genome structure.

First, there do not seem to be any TADs in Figure I.3 (they would create a prominent red triangle pattern along the diagonal) which would indicate that Sds3 regulates chromatin domains (Tanizawa et al., 2017). This means that Sds3 is not necessary for the creation and regulation of chromatin domains.

However, Figure I.3 shows that Sds3 is important in regulating several interchromosome interactions, most prominently that of the mating type locus as shown in blue on the heat map. This is also indicated by Figure III which shows more points deviating significantly from association trends in the control than in at other loci. This shows Sds3 has a role in keeping the mating type locus more transcriptionally active. The mating type locus is usually silenced by the presence of heterochromatin, but when this heterochromatin is removed due to starvation conditions, it allows for mating type switching and conjugation with cells of the opposite mating type (Arcangioli, 2000; Maki et al., 2018). My analysis showing Sds3 has a positive impact in regulating the mating type locus could mean that it has an important role in mating type switching. The involvement of components of HDAC 1 (such as Clr6) are recognized, but not fully understood (Thon et al., 2019). 

Though Sds3 does not have a significant role in regulating condensin domains, Figure II indicates that Sds3 has an important in regulating overall intrachromosome interactions. Sds3 is a component of HDAC I which preferentially deacetylates histones at specific intergenic regions that have promoters (Nicholas et al., 2007). Deacetylation results in more compact chromatin and therefore stronger intrachromosome interactions, so it makes sense that the lack of Sds3 would result in weaker intrachromosome interactions.

Finally, Figure III shows that the presence of Sds3 has a more prominent impact at loci with weaker association strength as shown by deviation from the diagonal trend in the lower left. This is likely because at lower interaction strengths, the same differences are magnified since it is a larger proportion of the whole as compared to at higher interaction strengths, making it seem as if there is a more significant deviation. This important fact shows just how prominent the deviation is at the mating type locus since the deviations there are at higher interaction strengths.

The limitations with this analysis include the fact that the heat map is not very versatile in that it is so large and such a great amount of data that it is difficult to determine loci without putting it through Juicer (a program used to more closely view and analyze Hi-C maps). Future directions could include isolating particular features of these heat maps for a deeper analysis. Additionally, these analyses can show general trends in chromosome interactions, but does not elucidate any mechanisms for these trends. This analysis could be built upon with further wet lab experiments that focus on Sds3's role at specific genomic loci such as the mating type locus. Finally, this research may not directly translate to that of human genome structure and disease since humans have more complex genome regulation mechanisms. This project can be further extended to humans by analyzing homologs to Sds3 in the human genome.

This study does not identify Sds3 as a regulator of condensin domains, but as a protein with a role in regulating overall intrachromosome organization, as well as interchromosome interactions at the mating type locus. The hypotheses are resolved, though exact mechanisms have not been elucidated and there may be further interactions not as easily distinguishable in this zoomed out map. These findings support previous findings about the role of Sds3 in HDAC I, as well as mating type switching.

My study contributes to the current field by eliminating Sds3 from the list of potential condensin interactors important in the regulation of chromatin domains. It also shows further evidence of the importance of Sds3 as a part of HDAC I in the regulation of overall intrachromosomal interactions. Finally, it reveals Sds3's importance in the regulation of mating type switching.

Overall, Sds3 is still essential for the highly organized eukaryotic 3D genome and the prevention of abnormal development and disease, even if not for chromatin domains.

## V. Literature Cited
Arcangioli, B. (2000). Fate of mat1 DNA strands during mating-type switching in fission yeast. EMBO Reports, 1(2). 145-150. https://dx.doi.org/10.1093%2Fembo-reports%2Fkvd023

Bahler, J., Wu, J., Longtine, M. S., Shah N. G., McKenzie, A., Steever, A. B., Wach, A., Phillippsen, P., & Pringle, J. R. (1998). Heterologous modules for efficient and versatile PCR-based gene targeting in Schizosaccharomyces pombe. Yeast Functional Analysis Reports, 14(10). 943-951. https://doi.org/10.1002/(SICI)1097-0061(199807)14:10<943::AID-YEA292>3.0.CO;2-Y

Dixon J. R., Selvaraj, S., Yue, F., Kim, A., Li, Y., Shen, Y., Hu, M., Liu, J. S., & Ren, B. (2012). Topological domains in mammalian genomes identified by analysis of chromatin interactions. Nature, 485(7398). 376-380. https://doi.org/10.1038/nature11082

Fields, S. & Song, O. (1989). A novel genetic system to detect protein-protein interactions. Nature, 340(6230). 245-246. https://doi.org/10.1038/340245a0

Hudson, D. F., Marshall, K. M., & Earnshaw, W. C. (2009). Condensin: Architect of mitotic chromosomes. Chromosome Research, 17. 131-144. https://doi.org/10.1007/s10577-008-9009-7

Kanke, M., Nishimura, K., Kanemaki, M., Kakimoto, T., Takahashi, T., Nakagawa, T., & Masukata, H. (2011). Auxin-inducible protein depletion system in fission yeast. BMC Cell Biology, 12(8). https://doi.org/10.1186/1471-2121-12-8

Lieberman-Aiden, E., van Berkum, N. L., Williams, L., Imakaev, M., Ragoczy, T., Telling, A., Amit, I., Lajoie, B. R., Sabo, P. J., Dorschner, M. O., Sandstrom, R., Bernstein, B., Bender, M. A., Groudine, M., Gnirke, A., Stamatoyannopoulos, J., Mirny, L. A., Lander, E. S., & Dekker, J. (2009). Comprehensive mapping of long range interactions reveals folding principles of the human genome. Science, 326(5950). 289-293. https://dx.doi.org/10.1126%2Fscience.1181369

Maki, T., Ogura, N., Haber, J. E., Iwasaki, H., & Thon, G. (2018). New insights into donor directionality of mating-type switching in Schizosaccharomyces pombe. PLoS Genetics, 14(5). https://doi.org/10.1371/journal.pgen.1007424

Nasmyth, K., Haering, C. H. (2009). Cohesin: its roles and mechanisms. Annual Review of Genetics, 43(1). 525-558. https://doi.org/10.1146/annurev-genet-102108-134233

Nicholas, E., Yamada, T., Cam, H. P., FitzGerald, P. C., Kobayashi, R., & Grewal, S. (2007). Distinct roles of HDAC complexes in promoter silencing, antisense suppression and DNA damage protection. Nature Structural & Molecular Biology, 14. 372-380. https://doi.org/10.1038/nsmb1239

Noma, K. (2017). The yeast genomes in three dimensions: mechanisms and functions. Annual Review of Genetics, 27(51). 23-44. https://doi.org/10.1146/annurev-genet-120116-023438

Simonis, M., Klous, P., Splinter, E., Moshkin, Y., Willemsen, R., de Wit, E., van Steensel, B., & de Laat, W. (2006). Nuclear organization of active and inactive chromatin domains uncovered by chromosome conformation capture-on-chip (4C). Nature Genetics, 38(11). 1348-1354. https://doi.org/10.1038/ng1896

Tanizawa, H., Kim, K., Iwasaki, O., & Noma, K. (2017). Architectural alterations of the fission yeast genome during the cell cycle. Nature Structural & Molecular Biology, 24(11). 965-976. https://doi.org/10.1038/nsmb.3482

Thon, G., Maki, T., Haber, J. E., & Iwasaki, H. (2019). Mating-type switching by homology-directed recombinational repair: a matter of choice. Current Genetics, 65. 351-362. https://doi.org/10.1007/s00294-018-0900-2

Zheng, H. & Xie, W. (2019). The role of 3D genome organization in development and cell differentiation. Nature Reviews Molecular Cell Biology, 20. 435-550. https://doi.org/10.1038/s41580-019-0132-4


